### ISSUE TITLE
TinyMCE: let users set the title in file/image upload

### ISSUE THREAD

**Issue by cekk** [plone/mockup#796]
In old versions of Tiny (on Plone 4), the upload form for files and images allows users to set also the title and description for the uploaded content.

In mockup's version of tiny, this isn't possible. What do you think about re-enable it?

btw i'm trying to fix that issue by myself, but i don't know exactly where to do these kind of things.
I've found the [template](https://github.com/plone/mockup/blob/master/mockup/patterns/upload/templates/upload.xml) used in the upload popup, but if i try to change it, nothing happens. Should i need to re-build mockup?

**Comment by vincentfretin** [comment]
You're talking about image caption right? This is now on Plone 5.2.2 to be released. You can close if it's ok for you.

**Comment by cekk** [comment]
nope, i was talking about to set the title and description fields for uploaded content (File or Image) that will be created
